cla_public/libs/mailgun.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from flask import current_app
+
+
+def send_ref_by_email(to, subject, email_body):
+    return requests.post(
+        'https://api.mailgun.net/v3/{domain}/messages'.format(
```

Comment:
I don't imagine the API will change much, but the version number in the URL might. I guess it doesn't matter that much. But most of our API URLs are in config - it seems better to keep them together if you need to find them.